### PR TITLE
feat (sdk): deprecated routes

### PR DESCRIPTION
**Pull request by yesnault**
Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

File: sdk/cdsclient/deprecated.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	}
+
+	return nil
+}
+
+func (c *client) ApplicationPipelineTriggerAdd(t *sdk.PipelineTrigger) error {
+	uri := fmt.Sprintf("/project/%s/application/%s/pipeline/%s/trigger", t.SrcProject.Key, t.SrcApplication.Name, t.SrcPipeline.Name)
+
+	if t.SrcEnvironment.Name != "" {
+		uri = fmt.Sprintf("%s?env=%s", uri, t.SrcEnvironment.Name)
```
Comment (bnjjj) [inline sdk/cdsclient/deprecated.go]: queryEscape

File: sdk/cdsclient/deprecated.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	}
+
+	return nil
+}
+
+func (c *client) ApplicationPipelineTriggersGet(projectKey string, appName string, pipelineName string, envName string) ([]sdk.PipelineTrigger, error) {
+	uri := fmt.Sprintf("/project/%s/application/%s/pipeline/%s/trigger/source", projectKey, appName, pipelineName)
+
+	if envName != "" {
+		uri = fmt.Sprintf("%s?env=%s", uri, envName)
```
Comment (bnjjj) [inline sdk/cdsclient/deprecated.go]: queryEscape

File: sdk/cdsclient/deprecated.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+func (c *client) ApplicationPipelineTriggerAdd(t *sdk.PipelineTrigger) error {
+	uri := fmt.Sprintf("/project/%s/application/%s/pipeline/%s/trigger", t.SrcProject.Key, t.SrcApplication.Name, t.SrcPipeline.Name)
+
+	if t.SrcEnvironment.Name != "" {
+		uri = fmt.Sprintf("%s?env=%s", uri, t.SrcEnvironment.Name)
+	}
+
+	app := &sdk.Application{}
```
Comment (fsamin) [inline sdk/cdsclient/deprecated.go]: why get the application from the response but return nil ?

Comment (yesnault) [reply sdk/cdsclient/deprecated.go]: yes, no need (as old sdk). Removed.

**Pull request merged**